<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>providejob</class>
 <widget class="QDialog" name="providejob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_update">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_jhlogo_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_jhlogo_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_update">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Provide Job</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pjpf">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_up">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>1261</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_cancelpjob">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_jn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Job Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ja">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Job Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_qf">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualification</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_salary">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_jobt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Job Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ja">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_jn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_qf">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_salary">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_jobt">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_post">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Job</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_desc">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
